website/configs/plugins.js

```diff
@@ -1,42 +1,38 @@
 const path = require('path')
 const setupPlugins = function ({ root, docPath }) {
-    const plugins = [
-        '@docusaurus/plugin-content-pages',
-        [
-            path.resolve(root, 'src/plugins/docusaurus-plugin-content-docs-extend'),
```

Comment:
the previous design it to let dev engineer take minimal job to onboard a demo. so I add this plugin to take all the front matters in readme file of each demo to auto onboard demo:
* easy config: the front matter is a section with less than 10 lines. the demo author only need to understand front matters in readme file of his/her demo.
* easy maintaintence: no need to maintain a doc for website and a readme in demo folder


but now, onboard a demo to the website is more difficult:
* requires knowledges:
  * `react`: in order to add readme
  * `mdx`: in order to add readme
  * `DataDemos.js`: in order to onboard a demo
* configs severals files:
  * add a config in website/src/DataDemos.js
  * add a readme only for the website e.g. `website/docs/realtime_scoreboard.mdx`
  * you need to author a component to show the doc `website/src/components/DemoPage/RealtimeScoreboard/RealtimeScoreboard.js`
* more maintaintence work: you need to maintain both `RealtimeScoreboard.js` and `xxx/scorebaord/readme.md` if you want to update

Reply:
if demo author need to write a react component for `mdx` file, and we don't lererage the core feature to auto convert markdown file to html of the framework, I don't understand why we need to update the demo site like this?

Reply:
What I'd like to achieve is for each demo there's an "overview", "run locally", "deploy to Azure", "resources" sections. From a site visitor's perspective, it's easier for them to navigate the site and browser the demos. 

However, currently when the demos were created, the format is very different since it's created by different devs without a standard format. What I am trying to do as a temporary solution is to handwrite react components for now and make the site live first. Over time, it's definitely a must to use the auto convert feature especially when we have lots of demos. 

Please let me know if I have explained myself well.